src/cp_parallel/cp_parallel.py

```diff
@@ -0,0 +1,153 @@
+"""
+This collection of functions runs CellProfiler in parallel at the plate level and can convert the results into log files
+for each process.
+"""
+
+import logging
+import multiprocessing
+import os
+import pathlib
+import subprocess
+
+from concurrent.futures import Future, ProcessPoolExecutor
+from logging import FileHandler, Formatter
+from typing import List
+
+from errors.exceptions import MaxWorkerError
+
+
+def results_to_log(
+    results: List[subprocess.CompletedProcess], log_dir: pathlib.Path, run_name: str
+) -> None:
```

Comment:
This will be in reference to issue #4. I currently don't see an exact benefit to returning something when the CellProfiler command is what does the returning.